src/views/stake/Stake.vue

```diff
@@ -185,6 +185,24 @@ export default {
       return store.getters["application/balance"].seen
     })
 
+    const initialize = async () => {
```

Comment:
had to move the function as it threw an `initialize` not found error in the console. Have to define it before it's called.